src/bsoncxx/config/postlude.hpp

```diff
@@ -11,3 +11,35 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
```

Comment:
given that most of these macros are defined by CMake, it would be greatly preferable if we could generate the postlude automatically from CMake. Can you investigate if there is a feasible way to do this, at least for the macros generated by CMake?